src/infra/docs/bastion.md

```diff
@@ -12,20 +12,22 @@ directly to a production server with SSH. Instead, all connections must come
 from a small server called the "bastion", which only allows connections from a
 few whitelisted networks and logs any connection attempt.
 
-To log into a server through the bastion you can use SSH's `-J` flag:
-
-```
-ssh -J bastion.infra.rust-lang.org servername.infra.rust-lang.org
-```
-
-It's also possible to configure SSH to always jump through the bastion when
-connecting to a host. Add this snippet to your SSH configuration file (usually
-located in `~/.ssh/config`):
-
-```
-Host servername.infra.rust-lang.org
-    ProxyJump bastion.infra.rust-lang.org
-```
+To log into a server through the bastion, use one of the following methods:
+
+- use SSH's `-J` flag:
+  ```
+  ssh -J <username>@bastion.infra.rust-lang.org <username>@servername.infra.rust-lang.org
+  ```
```

Comment:
Nit: I would follow the convention in other parts of the document to surround code blocks with empty lines. Some for the other code blocks a bit further down...

Reply:
I formatted the entire document with prettier 👍